lib/restbase.js

```diff
@@ -164,6 +165,9 @@ RESTBase.prototype.defaultListingHandler = function(value, restbase, req) {
 
 // Special handling for external web requests
 RESTBase.prototype.defaultWebRequestHandler = function(req) {
+    // enforce the usage of UA
+    req.headers = req.headers || {};
+    req.headers['User-Agent'] = req.headers['User-Agent'] || this.rb_config.user_agent;
```

Comment:
> I agree consistency matters, but the larger question is: should we Correctly-Capitalise all of the outgoing headers? It's a standard to do so after all.

Unless there is a compelling reason to lower-case them, I think we should honor the standard (even if it means opening a separate issue to fix existing lower-cased headers).